tractolearn/tractoio/dataset_fetch.py

```diff
@@ -0,0 +1,528 @@
+# -*- coding: utf-8 -*-
+
+import contextlib
+import enum
+import os
+import subprocess
+import sys
+import tarfile
+import zipfile
+from hashlib import md5
+from os.path import join as pjoin
+from shutil import copyfileobj
+from urllib.request import urlopen
+
+
+TRACTOLEARN_DATASETS_URL = "https://zenodo.org/record/"
+
+key_separator = ","
+
+
+class Dataset(enum.Enum):
+    """Datasets for tractography learning.
+    """
+
+    CONTRASTIVE_AUTOENCODER_WEIGHTS = "contrastive_ae_weights"
+    MNI2009CNONLINSYMM_ANAT = "mni2009cnonlinsymm_anat"
+    TRACTOINFERNO_HCP_REF_TRACTOGRAPHY = "tractoinferno_hcp_ref_tractography"
+    BUNDLE_LABEL_CONFIG = "bundle_label_config"
+    TRACTOINFERNO_HCP_CONTRASTIVE_THR_CONFIG = "tractoinferno_hcp_contrastive_thr_config"
+    GENERATIVE_STRML_MAX_COUNT_CONFIG = "generative_streamline_max_count_config"
+    GENERATIVE_STRML_RQ_COUNT_CONFIG = "generative_streamline_req_count_config"
+    GENERATIVE_SEED_STRML_RATIO_CONFIG = "generative_seed_streamline_ratio_config"
+    GENERATIVE_LOA_CONE_CONFIG = "generative_loa_cone_config"
+    GENERATIVE_WM_TISSUE_CRITERION_CONFIG = "generative_wm_tisue_criterion_config"
+
```

Comment:
Can you fetch and unzip François atlas: `atlas.zip` -> https://zenodo.org/record/7562635#.Y-EXWNLMJH4

Reply:
Done in 063e1d3.